messenger.rst

```diff
@@ -839,11 +839,73 @@ Transport Configuration
 -----------------------
 
 Messenger supports a number of different transport types, each with their own
-options.
+options. Options can be passed to the transport via a DSN string or configuration.
+
+.. code-block:: bash
+
+    # .env
+    MESSENGER_TRANSPORT_DSN=amqp://localhost/%2f/messages?auto_setup=false
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/messenger.yaml
+        framework:
+            messenger:
+                transports:
+                    my_transport:
+                        dsn: "%env(MESSENGER_TRANSPORT_DSN)%"
+                        options:
+                            auto_setup: false
+
+    .. code-block:: xml
+
+        <!-- config/packages/messenger.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony
+                https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:messenger>
+                    <framework:transport name="my_transport" dsn="%env(MESSENGER_TRANSPORT_DSN)%">
+                        <framework:options>
+                            <framework:auto-setup>false</framework:auto-setup>
+                        </framework:options>
+                    </framework:transport>
+                </framework:messenger>
+            </framework:config>
+        </container>
```

Comment:
Please double check this. I chanced it slightly because I think the current config was incorrect. 

Here is what I did (basically):

```diff
 <framework:config>
        <framework:messenger>
            <framework:transport name="async_priority_high" dsn="%env(MESSENGER_TRANSPORT_DSN)%?queue_name=high_priority"/>
            <framework:transport name="async_priority_low" dsn="%env(MESSENGER_TRANSPORT_DSN)%">
                <framework:options>
-                   <framework:queue>
-                        <framework:name>normal_priority</framework:name>
-                    </framework:queue>
+                   <framework:queue-name>normal_priority</framework:name>
                </framework:options>
            </framework:transport>
        </framework:messenger>
    </framework:config>
```